pubsub/cloud-client/publisher.py

```diff
@@ -131,7 +131,9 @@ def publish_messages_with_error_handler(project, topic_name):
     topic_path = publisher.topic_path(project, topic_name)
 
     def callback(message_future):
-        if message_future.exception():
+        # Attempts to publish for 10 seconds, then times out.
+        # Defaults to None.
```

Comment:
What's the behavior when timeout is None?

This comment should be more specific. Something like: When timeout is unspecified or None, the exception method waits indefinitely.

Reply:
That's much better. Let me rewrite it.